chrome_manifest_v3/views/popupView.js

```
@@ -1098,26 +1112,26 @@ Gmail2Trello.PopupView.prototype.bindGmailData = function (data = {}) {
-    var img = '';
-    var img_big = '';
+    let html = '';
+    let img = '';
+    let img_big = '';
     const domTag_k = `#g2t${tag.charAt(0).toUpperCase()}${tag
       .slice(1)
       .toLowerCase()}`;
-    var $domTag = $(domTag_k, self.$popup);
+    const $domTag = $(domTag_k, self.$popup);
 
     const domTagContainer = domTag_k + 'Container';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)